workflows/phylogenetics/wf_nextclade_addToRefTree.wdl

```diff
@@ -8,28 +8,22 @@ workflow nextclade_addToRefTree {
       description: "Nextclade workflow that adds samples to a curated JSON tree from Augur."
     }
     input {
-      File assembly_fastas
-      File? root_sequence_fasta
```

Comment:
Same comment as above RE input rename

Reply:
resolved in [0800fa0](https://github.com/theiagen/public_health_bioinformatics/pull/375/commits/0800fa0ce76b0faead325227b1ec2cda428361c9)